tensorflow/lite/micro/kernels/ceva/CEVA_basic_op.h

```diff
@@ -0,0 +1,520 @@
+
+
+#ifndef _CEVA_BASIC_OP_H
+#define _CEVA_BASIC_OP_H
+typedef int I32;
+typedef short I16;
+// typedef int		I32;
+
+#ifndef MAX_32
+#define MAX_32 (I32)0x7fffffffL
+#endif
+
+#ifndef MIN_32
+#define MIN_32 (I32)0x80000000L
+#endif
+
+#ifdef CEVA_SensPro
+static inline I32 L_mpy_ll_sat_rnd(I32 inp1, I32 inp2);
+
+static I32 L_shl(I32 L_var1, I16 var2);
+static I32 L_shr(I32 L_var1, I16 var2);
+
+static I32 L_shr(I32 L_var1, I16 var2) {

```

Comment:
Prefer inline to static in the header?

If we have static (without the inline), we will certainly get code bloat without the benefits of inlining?

And if we have `static inline` then we can still get code bloat (if compiler decides to not inline) without any performance gains?

Only inline in the header seems to me to be the case where we either get performance with increase in code size (if the compiler does indeed inline), or shared code so no unnecessary code size increase (if the compiler decides to not inline).

WDYT?

Reply:
Removed dependency on this file, not needed anymore